app/views/bulk_actions/index.html.erb

```diff
@@ -1,7 +1,14 @@
 <div class='container'>
 
   <h1>Bulk Actions</h1>
-  <%= link_to 'New Bulk Action', new_bulk_action_path, class: 'btn btn-success' %>
+  <div>
+    <div style="float: left">
```

Comment:
Why does this need to float?  Can you use the grid layout instead?

Reply:
Sure, this was quick, but look look at a grid layout.